README.md

```diff
@@ -64,7 +79,15 @@ argument. Finally, demonstrate calling `doSomething` with a function.**
 
 Your Answer:
 ```js
-// write code here
+function doSomething(thingToDo){
+  return thingToDo();
+}
+
+function thingToDo(){
+  alert("I was done");
+}
```

Comment:
Looks good.  Recommend naming this function something other than the argument name, to avoid confusion.